docs/manage-model-metadata-using-gravitino.md

```diff
@@ -0,0 +1,637 @@
+http://localhost:8090/api/metalakes/example/catalogs/model_catalog/schemas/model_schema/models
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// ...
+Catalog catalog = gravitinoClient.loadCatalog("model_catalog");
+NameIdentifier[] identifiers = catalog.asModelCatalog().listModels(Namespace.of("model_schema"));
+// ...
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+gravitino_client: GravitinoClient = GravitinoClient(uri="http://localhost:8090", metalake_name="example")
+
+catalog: Catalog = gravitino_client.load_catalog(name="model_catalog")
+model_list = catalog.as_model_catalog().list_models(namespace=Namespace.of("model_schema")))
+```
+
+</TabItem>
+</Tabs>
+
+## Model version operations
+
+:::tip
+ - Users should create a metalake, a catalog, a schema, and a model before link a model version
+   to the model.
+:::
+
+### Link a model version
+
+You can link a model version by sending a `POST` request to the `/api/metalakes/{metalake_name}
+/catalogs/{catalog_name}/schemas/{schema_name}/models/{model_name}` endpoint or by using
+the Gravitino Java/Python client. The following is an example of linking a model version:
+
+<Tabs groupId="language" queryString>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
+-H "Content-Type: application/json" -d '{
+  "uri": "path/to/model",
+  "aliases": ["alias1", "alias2"],
+  "comment": "This is version 0",
+  "properties": {
+    "k1": "v1"
+  }
+}' http://localhost:8090/api/metalakes/example/catalogs/model_catalog/schemas/model_schema/models/example_model
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// ...
+Catalog catalog = gravitinoClient.loadCatalog("model_catalog");
+catalog.asModelCatalog().linkModelVersion(
+    NameIdentifier.of("model_schema", "example_model"),
+    "path/to/model",
+    new String[] {"alias1", "alias2"},
+    "This is version 0",
+    ImmutableMap.of("k1", "v1"));
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python
+gravitino_client: GravitinoClient = GravitinoClient(uri="http://localhost:8090", metalake_name="example")
+
+catalog: Catalog = gravitino_client.load_catalog(name="model_catalog")
+catalog.as_model_catalog().link_model_version(model_ident=NameIdentifier.of("model_schema", "example_model"),
+                                              uri="path/to/model",
+                                              aliases=["alias1", "alias2"],
+                                              comment="This is version 0",
+                                              properties={"k1": "v1"})
+```
+
+</TabItem>
+</Tabs>
+
+The comment and properties of model version can be different from the model.
+
+### Get a model version
+
+You can get a model version by sending a `GET` request to the `/api/metalakes/{metalake_name}
+/catalogs/{catalog_name}/schemas/{schema_name}/models/{model_name}/versions/{version_number}`
```

Comment:
Okay, here it is ... `/api/.../models/{model}/versions/{version}` is more RESTful than the previous API. We really should POST to the `/api/.../models/{model}/versions` URI for "creating" a ModelVersion resource then.